scripts/travis/apt-install

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e -x
+
+export DEBIAN_FRONTEND=noninteractive
+
+install_retry_counter=0
+max_apt_retries=5
+
+# This function loops a couple of times over apt-get, hoping to
+# avoid CI errors due to errors during apt-get
+# hashsum mismatches, DNS errors and similar things
+while true; do
+	sleep ${install_retry_counter}
```

Comment:
sleep has to be after apt-get install

Reply:
I moved here so that even after:
```
	apt-get clean -qqy || continue
	apt-get update -qqy || continue
```
there will be a sleep. Especially if one of the commands fails because of temporary network problems, a sleep might help to resolve those problems. At least that is one of the ideas behind this PR. If the sleep is at the end it will only run if `apt-get install` fails not if `clean` (okay, really unnecessary) or `update` fails. Therefore I moved it to the beginning. As the `install_retry_counter` is zero it will not introduce a delay for the first run if everything works.

Reply:
oh. you call sleep with the zero timeout. I agree, this will work, but I think the code will look better and will be more readable if we move this sleep after apt calls and do this change:

apt-get clean -qqy && apt-get update -qqy && apt-get install -qqy --no-install-recommends $@ && break

Let me know if you don't think so and I will merge it as is.

Reply:
Will
```
apt-get clean -qqy && apt-get update -qqy && apt-get install -qqy --no-install-recommends $@ && break
```
work in combination with `set -e`? Or will the `set -e` abort the whole script?

Reply:
I looked at the documentation and `set -e` does the right thing here. I changed it to be what you suggested.